processing/src/main/java/org/apache/druid/segment/nested/NestedDataColumnSupplier.java

```diff
@@ -91,6 +95,15 @@ public NestedDataColumnSupplier(
         fields = GenericIndexed.read(bb, GenericIndexed.STRING_STRATEGY, mapper);
```

Comment:
This is a comment on old code and I think we'd have to push the version forward yet again to adjust this, which might not be worth it, but it's a bit sad to me that we aren't using the front-coded stuff for the fields.  They are pretty much guaranteed to benefit from it.

Reply:
yeah, will keep that in mind the next time we need to bump format version